src/main/webapp/index.html

```diff
@@ -4,4 +4,16 @@
 <title>ZoomTube</title>
 <link rel="stylesheet" href="style.css" />
 <script src="script.js"></script>
-<p>Hello world!</p>
+<form name="lectureForm" action="/lecture" method="POST" onsubmit="return validateForm()" required>
```

Comment:
I think `required` belongs in the `input` element, then you don't need the javascript.  It should make the thing red and say input required automatically if you try to submit with it empty.  See: https://www.w3schools.com/tags/att_input_required.asp

Reply:
you're right, fixed